src/Sdk/DTWebApi/WebApi/ActionsStepTelemetry.cs

```diff
@@ -56,5 +56,7 @@ public ActionsStepTelemetry()
 
         [DataMember(EmitDefaultValue = false)]
         public int? ExecutionTimeInSeconds { get; set; }
+        [DataMember(EmitDefaultValue = false)]
+        public bool IsContainerHookStep { get; set; }
```

Comment:
is this used anywhere?

Reply:
Will be, it's in a PR against the fork@main